utils/librarypath/librarypath.go

```diff
@@ -39,6 +40,25 @@ func cleanPath(paths []string, fullPath bool) []string {
 	return processedPaths
 }
 
+func ExistsInArray(array []string, item string) bool {
```

Comment:
You can use https://github.com/HashLoad/boss/blob/ed9aae1b2d3a9583e62629ae58e023153629c98c/utils/arrays.go#L5